README.md

```diff
@@ -33,7 +33,7 @@ Clone the Git repository and run the following commands:
 npm install
 export DISCORD_TOKEN="<your Discord bot token>"
 export GOOGLE_TOKEN="<your Google (Maps) API key>"
-export CR_ADDRESS="https://media.wizards.com/2020/downloads/MagicCompRules%2020200417.txt"
+export CR_ADDRESS="https://media.wizards.com/2020/downloads/MagicCompRules%2020200925.txt"
```

Comment:
Actually, this would be a better default now: 
```suggestion
export CR_ADDRESS="http://cr.vensersjournal.com"
```
https://github.com/bra1n/judgebot/pull/141/files#diff-d4f5e35b892d3603ca531ef9da4b2964R8

Reply:
Ah great, I didn't realise you'd done that. In that case, should we not just remove most of these env. variables from the usage, since the default behaviour already does this?

Reply:
Fair enough, suits me well. :-)